cob_bringup/robots/cob4-7.xml

```diff
@@ -44,7 +44,6 @@
 		</include>
 		<include file="$(find cob_bringup)/drivers/bms.launch">
 			<arg name="robot" value="$(arg robot)"/>
-			<arg name="sim" value="true"/>
```

Comment:
we need this line to be `<arg name="sim" value="$(arg sim)"/>` otherwise we'll break simulation!